src/main/java/hudson/plugins/jobConfigHistory/JobConfigHistoryJobListener.java

```diff
@@ -137,5 +147,12 @@ public void deleteItem(Item item) {
 					"onDeleted: not an AbstractItem {0}, skipping.", item);
 		}
 
+		@Override
+		public void changeItemLocation(Item item, String oldFullName, String newFullName) {
+			//TODO implement
```

Comment:
Something to do here? Otherwise remove this line.

Reply:
Done.